src/olympia/blocklist/tasks.py

```diff
@@ -30,25 +30,30 @@ def create_blocks_from_multi_block(multi_block_submit_id, **kw):
 @task
 @use_primary_db
 def import_block_from_blocklist(record):
-    log.debug('Processing block id: [%s]', record.get('id'))
+    kinto_id = record.get('id')
+    log.debug('Processing block id: [%s]', kinto_id)
+    kin = KintoImport.objects.create(kinto_id=kinto_id, record=record)
```

Comment:
This seems like an odd name for this variable (being so short and not entirely obvious that it corresponds to a `KintoImport` object), but if this matches the standards for variable naming in the project then I guess it's fine.

Reply:
https://github.com/mozilla/addons-server/pull/13274/commits/05b0c2536b4bf1b1f310a482ad84345acf39477d